docker/Dockerfile

```diff
@@ -0,0 +1,107 @@
+FROM --platform=linux/amd64 node:16.13.0 AS frontend
+
+WORKDIR /app/oppia
+
+RUN apt-get update -y && apt-get upgrade -y \
+    curl \
+    git \
+    npm
+
+# Installing packages from the package.json file
+COPY package.json .
+COPY scripts/linters/custom_eslint_checks ./scripts/linters/custom_eslint_checks
+
+RUN npm install -g yarn@1.22.15 --force
+# mathjs ESOCKETTIMEDOUT error - there are cases reported for mathjs package when timeout happens while
+# downloading the package using `yarn install` command.
+# The solution is to increase the network timeout, so that the package is downloaded successfully.
+RUN yarn install --network-timeout 1000000
```

Comment:
Have you faced the issue mentioned here? DO we do this in start.py as well?

Reply:
yes, I faced it in the docker setup. but not within the python setup. so lets keep this only in docker setup. is that fine?

Reply:
no need to timeout, I removed and tested it